packages/editor-ui/src/stores/templates.store.ts

```diff
@@ -320,12 +311,13 @@ export const useTemplatesStore = defineStore(STORES.TEMPLATES, {
 			try {
 				const payload = await getWorkflows(apiEndpoint, {
 					...query,
-					skip: cachedResults.length,
+					skip: cachedResults.length / TEMPLATES_PAGE_SIZE + 1,
```

Comment:
is this not a skip value anymore? maybe we should rename these props to make sense then.

Reply:
Agree, I wanted to keep the interface the same but since the endpoint is new, makes sense to rename